src/cplscheme/BaseCouplingScheme.hpp

```diff
@@ -140,21 +140,13 @@ class BaseCouplingScheme : public CouplingScheme {
    */
   double getTimeWindowSize() const override final;
 
-  /**
-   * @brief Returns the remaining timestep length within the current time window.
-   *
-   * If no time window size is prescribed by the coupling scheme, always 0.0 is
```

Comment:
Returning 0.0 here I think is not a good idea (looking at the behavior of `method="participant-first"`